src/lib/to_graph.ml

```diff
@@ -1,14 +1,52 @@
 (** Construction from Mas_parser *)
```

Comment:
For posterity, this is just github saving this comment on the old file that was replaced by the new one.